src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -134,4 +134,15 @@ ResponseEntity clearCachedTemplateForCaseType(@PathVariable String caseType) {
         caseDataService.clearCachedTemplateForCaseType(caseType);
         return ResponseEntity.ok("Cache Cleared");
     }
+
+    @GetMapping(value = "/case/{caseUUID}/data/{variableName}")
+    ResponseEntity<String> getCaseDataValue(@PathVariable UUID caseUUID, @PathVariable String variableName) {
+        return ResponseEntity.ok(caseDataService.getCaseDataField(caseUUID, variableName));
+    }
+
+    @PutMapping(value = "/case/{caseUUID}/data/{variableName}")
+    ResponseEntity updateCaseDataValue(@PathVariable UUID caseUUID, @PathVariable String variableName, @RequestBody String value) {
+        caseDataService.updateCaseData(caseUUID, null, Map.of(variableName, value));
```

Comment:
Will passing NULL into Stage UUID lose auditing, and reporting?

Reply:
I was worried about it as well. From my testing it looked like there was no impact at all. I'm not passing a stage uuid here as the update takes place in between 2 stages (on top level workflow). This is also why i added new endpoint that does not verify that a user is assigned to a case. Thorough testing of reporting mechanism will be performed in qa as part of this jira.